R/fotmob_leagues.R

```diff
@@ -303,11 +303,8 @@ fotmob_get_league_matches <- function(country, league_name, league_id, season =
     season = season
   )
   .fotmob_message_for_season(resp, season)
-  rounds <- resp$matches$data$matchesCombinedByRound
-  purrr::map_dfr(
```

Comment:
in addition to renaming the JSON element, it's no longer returned as a list, so there is no more need for a `map_dfr` here (the `roundName` is guaranteed to only have one type if there is just one data frame)